util/plugins/eu.esdihumboldt.util.groovy.sandbox/META-INF/MANIFEST.MF

```diff
@@ -9,6 +9,7 @@ Require-Bundle: groovy;bundle-version="2.5.19",
 Export-Package: eu.esdihumboldt.util.groovy.sandbox,
  eu.esdihumboldt.util.groovy.sandbox.internal;x-internal:=true
 Import-Package: de.fhg.igd.eclipse.util.extension,
+ eu.esdihumboldt.hale.common.core.service,
```

Comment:
Instead of adding a dependency here, the allowed class should be added via the [extension point](https://github.com/halestudio/hale/blob/85dfae5fc2979cc885fd511f267874a49045fe44/util/plugins/eu.esdihumboldt.util.groovy.sandbox/plugin.xml#L4) in the bundle where the `DelegateServiceProvider` class is defined.

See [here](https://github.com/halestudio/hale/blob/d56e083333fd46f309c9088eee4820de9f44ae54/common/plugins/eu.esdihumboldt.hale.common.align/plugin.xml#L639) for an example.